todos_test.go

```diff
@@ -13,21 +13,16 @@ func TestListTodos(t *testing.T) {
 
 	mux.HandleFunc("/api/v4/todos", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		fmt.Fprint(w, `[{"id":1,"state": "pending","target":{"id":1,"approvals_before_merge":2}},{"id":2,"state":"pending","target":{"id":2,"approvals_before_merge":null}}]`)
+		mustWriteHTTPResponse(t, w, "testdata/list_todos.json")
 	})
 
-	opts := &ListTodosOptions{}
+	opts := &ListTodosOptions{ListOptions: ListOptions{PerPage: 2}}
```

Comment:
I'm aware this doesn't change the fixture, nor it's output. But it does test the interface.